### DISCUSSION TITLE
Docker and rest problem?

### DISCUSSION THREAD

**Discussion by rlljorge** [ltb-project/self-service-password#995] [General]
I am using the Docker version tag 1.7, and the Apache server is missing a configuration:

```Alias /rest /var/www/rest``` 

After manually adding the Alias, another issue appears in an include:

```PHP Fatal error:  Uncaught Error: Failed opening required '../../lib/detectbrowserlanguage.php' (include_path='.:/usr/local/lib/php') in /var/www/rest/v1/include.php:24\nStack trace:\n#0 /var/www/rest/v1/checkpassword.php(2): require_once()\n#1 {main}\n  thrown in /var/www/rest/v1/include.php on line 24```

Any ideas?

Rodrigo

**Comment by coudot** [comment]
This is a bug, please open an issue
